phpunit.xml.dist

```diff
@@ -4,7 +4,6 @@
         xsi:noNamespaceSchemaLocation="tools/phpunit/vendor/phpunit/phpunit/phpunit.xsd"
         bootstrap="vendor/autoload.php"
         colors="true"
-        cacheResultFile="./var/phpunit/cache/phpunit.cache"
```

Comment:
hmm why?

Reply:
This option does not exist anymore. See https://github.com/sebastianbergmann/phpunit/issues/4600